x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_attributes/users/kpi_user_authentications_area.ts

```diff
@@ -15,7 +15,7 @@ export const kpiUserAuthenticationsAreaLensAttributes: LensAttributes = {
   state: {
     visualization: {
       axisTitlesVisibilitySettings: {
-        x: true,
```

Comment:
This change in sneaking into another PR hehehe

Reply:
Ahhh! Sorry about that!!